Core/tests/Unit/EmulatorTraitTest.php

```diff
@@ -22,6 +22,8 @@
 use Google\Cloud\Core\Testing\TestHelpers;
 use PHPUnit\Framework\TestCase;
 
+use function extension_loaded;
+
```

Comment:
nit: just add a `\` before the function to achieve this (see below)...

```suggestion
use function extension_loaded;

```